### PR TITLE
Mercury 13.11.0 plat 8047

### PR DESCRIPTION
**Pull request by noam-arad**

### CODE REVIEW

File: api_v3/services/ServerNodeService.php [hunk 1]
```diff
@@ -188,4 +190,26 @@ private function addNewServerNode(KalturaServerNode $serverNode)
...
+	 *
+	 * @action unregister
+	 * @param string $serverNodeId
+	 * @throws KalturaErrors::INVALID_OBJECT_ID
+	 * @return KalturaServerNode
+	 * @throws KalturaAPIException
+	 */
+	function unregisterAction($serverNodeId)
+	{
+		$dbServerNode = ServerNodePeer::retrieveByPK($serverNodeId);
```
Comment (yossipapi) [inline api_v3/services/ServerNodeService.php]: You should probably use retrieveActiveServerNode to not fetch serverNodes which are disabled.

File: api_v3/services/ServerNodeService.php [hunk 2]
```diff
@@ -192,17 +191,21 @@ private function addNewServerNode(KalturaServerNode $serverNode)
...
 	 * @param string $serverNodeId
 	 * @throws KalturaErrors::INVALID_OBJECT_ID
 	 * @return KalturaServerNode
 	 * @throws KalturaAPIException
 	 */
-	function unregisterAction($serverNodeId)
+	function markOfflineAction($serverNodeId)
 	{
-		$dbServerNode = ServerNodePeer::retrieveByPK($serverNodeId);
+		$criteria = new Criteria(ServerNodePeer::DATABASE_NAME);
```
Comment (yossipapi) [inline api_v3/services/ServerNodeService.php]: Why aren't U using ServerNodePeer::retrieveActiveServerNode?

File: deployment/updates/scripts/add_permissions/2018_10_01_server_node_unregister.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?php
+/**
```
Comment (yossipapi) [inline deployment/updates/scripts/add_permissions/2018_10_01_server_node_unregister.php]: Change the file name to match the new action you added.

File: api_v3/services/ServerNodeService.php (hunk 2 above)
Comment (noam-arad) [reply api_v3/services/ServerNodeService.php]: I have the serverNodeId and don't have the hostname

**Pull request merged**